__tests__/users.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/users", () => {
+  test("200 - GET - will return an article object with the given article_id, with the properties of author, title, article_id, body, topic, created_at, votes and article_img_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((res) => {
+        const users = res.body.users;
+        expect(users).toHaveLength(4);
+        expect(Array.isArray(users)).toBe(true);
+        users.forEach((user) => {
+          expect(user).toHaveProperty("username", expect.any(String));
+          expect(user).toHaveProperty("name", expect.any(String));
+          expect(user).toHaveProperty("avatar_url", expect.any(String));
+        });
+      });
+  });
+  test("404 - GET - sends an appropriate status and error message when an invalid URL is passed", () => {
```

Comment:
You have already tested for this, this end point cannot 404. All invalid endpoints will 404 and this is not unique to this end point.

Reply:
<img width="1003" alt="Screenshot 2024-08-27 at 14 22 17" src="https://github.com/user-attachments/assets/b99c47fc-90d8-4555-a980-8bfbec5f55a0">